server/model/monitor.js

```diff
@@ -613,6 +614,45 @@ class Monitor extends BeanModel {
 
                 } else if (this.type === "port") {
                     bean.ping = await tcping(this.hostname, this.port);
```

Comment:
Much better.
In terms of code, we are currently migrating to a new `monitor-types` structure instead of dumping everything in this file.

I would appreciate if you had the time to move this (and `tcping`) to its own monioring type.

https://github.com/louislam/uptime-kuma/blob/2e2acbd48af4aec981a7d47c9d52b1e80d26f86d/server/monitor-types/snmp.js